### PR TITLE
Add missing AchievementRepository bean def

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- close https://github.com/DroidKaigi/conference-app-2023/issues/1122

## Overview (Required)
- `AchievementRepository` bean definition is missing. So I added that.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

**Review by momomomo111 — APPROVED** [review]
I've also checked the functionality on the Android side.
LGTM!

File: app-ios/App/DroidKaigi2023/DroidKaigi2023.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -490,10 +491,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = DroidKaigi2023/DroidKaigi2023.entitlements;
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"DroidKaigi2023/Preview Content\"";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = DFY27HWN2G;
```
Comment (takahirom) [inline app-ios/App/DroidKaigi2023/DroidKaigi2023.xcodeproj/project.pbxproj:498]: I'm not sure if this is ok to merge. 👀

**Comment by tkhs0604** [comment]
Let me link this ticket https://github.com/DroidKaigi/conference-app-2023/issues/1122

File: app-ios/App/DroidKaigi2023/DroidKaigi2023.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ry-itto) [reply app-ios/App/DroidKaigi2023/DroidKaigi2023.xcodeproj/project.pbxproj:498]: Oh, this is not good. I'll remove this 🙏🏼

**Review by takahirom — APPROVED** [review]
👍

**Pull request merged**